exercises/calculator.cpp

```diff
@@ -10,3 +10,33 @@
   Multiplication: 8
   Division: 2
 */
+
+#include<iostream>
+
+using namespace std;
+
+int main(){
+  float num1;
+  float num2;
+
+  cout<<"Insert the first number: "<<endl;
+  cin>>num1;
+
+  cout<<"Insert the second number: "<<endl;
+  cin>>num2;
+
+  float sum,diff,mult,div;
+
+  sum = num1 + num2;
+  diff = num1 - num2;
+  mult = num1 * num2;
+  div = num1 / num2;
```

Comment:
Potresti wrappare questa parte in una funzione esterna in modo tale da poterla usare anche una seconda volta in un ipotetico futuro.